lightly/models/modules/masked_autoencoder_timm.py

```diff
@@ -1,15 +1,13 @@
 from __future__ import annotations
 
-import sys
 from functools import partial
-from typing import Callable, Literal, Optional, Tuple, Type, Union
+from typing import Callable, Optional
 
-try:
-    from timm.layers import LayerType, Mlp, PatchEmbed
+from lightly.utils import dependency
+
+if dependency.timm_vit_available():
     from timm.models import vision_transformer
```

Comment:
No need to add the check here. The import of the file will anyways fail if timm is not available.